src/ipyaladin/widget.py

```diff
@@ -294,6 +300,38 @@ def height(self, height: int) -> None:
         self._fov_xy = {}
         self._height = height
 
+    @property
+    def rotation(self) -> float:
+        """The center rotation of the widget.
+
+        This is the view center to north pole angle in degrees.
+        This is equivalent to getting the 3rd Euler angle.
+        Positive angles rotates the view in the counter clockwise
+        order (or towards the east).
+
+        It can be set with either a float number in degrees
+        or an astropy.coordinates.Angle object.
```

Comment:
documentation: this sentence would be better put in the setter doc string, since we aren't actually doing any setting in this method